### PR TITLE
Sec role timers

### PR DESCRIPTION
**Pull request by noudoit**
## About the PR 
- this PR aims to increase the playtime needed to become a cadet and sec officer along side an increase in other security roles. 
- this will also prevent fewer incompetent people instantly picking and joining sec as there is no time gate.
- cadets should have basic knowledge on how to play the game before being able to play cadet.
- with that being said Cadet time lock will be made 4 hours or 14400 seconds and officer will be made 2 hours or 7200 seconds and detective will also have the same requirement as officers.
**Media**

- [X ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- tweaked: time to become a cadet
- tweaked: time to become an officer
- working on: Warden and Det times to cater with officer and cadet changes + altering HOS requirements to work with these changes

### CODE REVIEW

**Comment by metalgearsloth** [comment]
> this PR aims to increase the playtime needed to become a cadet and sec officer .

It's not increasing playtime on cadet?

**Comment by noudoit** [comment]
> > this PR aims to increase the playtime needed to become a cadet and sec officer .
> 
> It's not increasing playtime on cadet?

i must have done something wrong (: hahhhah

**Comment by noudoit** [comment]
> I forgot other roles exist. Detective, warden, and HOS all use sec department role timers which should also be updated. Detective specifically is 3600, allowing you to become a detective before being allowed to play security officer. Warden is 10800 which would allow you to be a warden with possibly no secoff playtime.

what if i just made it require a certain role so to get any real role u need Sec officer playtime and det play time to unlock more such as warden and hos?

**Comment by EmoGarbage404** [comment]
idk, det playtime for those is really weird. I'd rather just keep it how it is. I'm not sure how playing as det is going to prove that you're a good HOS. I thought the purpose was just to prove that you have some knowledge of the department.

**Comment by noudoit** [comment]
I'm just coming up with extra ideas otherwise I'd just need to increase all roles to reasonable amounts

**Comment by noudoit** [comment]
> > In the changelog you can change `add` to `tweak` to indicate that this is a change to something existing and not new content
> 
> When I said this I meant the add before the `:` not the one after:
> 
> * add: Something new
> * tweak: Something change
> * fix: Something unborked
> * remove: Something gone

(: oh

**Comment by noudoit** [comment]
sorry if this takes a bit I'm extremely new to all this and hardly know what I'm looking at I'm just having fun trying to do something that may help. (:

File: Resources/Prototypes/Roles/Jobs/Security/security_cadet.yml [hunk 1]
```diff
@@ -6,8 +6,10 @@
   requirements:
     - !type:DepartmentTimeRequirement
       department: Security
-      time: 7200
+      time: 14400
       inverted: true # stop playing intern if you're good at security!
+    - !type:OverallPlaytimeRequirement
+      time: 14400
```
Comment (metalgearsloth) [inline Resources/Prototypes/Roles/Jobs/Security/security_cadet.yml:12]: Why does a cadet need 4 hours of playtime?

**Comment by metalgearsloth** [comment]
@noudoit you coming back to this?

File: Resources/Prototypes/Roles/Jobs/Security/security_cadet.yml (hunk 1 above)
Comment (QuietlyWhisper) [reply Resources/Prototypes/Roles/Jobs/Security/security_cadet.yml:12]: Probably to prevent entirely new players from playing sec.

**Comment by deltanedas** [comment]
should be revived first to add cadet playtime since that prevents brand new players from (unintentionally) harmbatoning as any sec role

**Pull request closed without merging**